packages/server/lib/db/migrations/20230510083713_activity_logging_table.cjs

```diff
@@ -0,0 +1,51 @@
+const tableName = '_nango_activity_logs';
+const messagesTableName = '_nango_activity_log_messages';
+
+exports.up = function (knex, _) {
+    return knex.schema
+        .withSchema('nango')
+        .createTable(tableName, function (table) {
+            table.increments('id').primary();
+            table.integer('account_id').unsigned().notNullable().index();
+            table.enu('level', ['error', 'warn', 'info', 'http', 'verbose', 'debug', 'silly']).defaultTo('info').notNullable();
+            table.enu('action', ['oauth', 'proxy', 'token', 'sync']).defaultTo('oauth').notNullable();
+            table.boolean('success');
+            table.bigInteger('timestamp');
```

Comment:
Isn't clearer/easier to store these 3 fields as timestamps?

Reply:
I was using the javascript Date.now() implementation so opted to keep this using that. Happy to revisit this after the initial merge